lib/fluent/plugin/in_cloudwatch_logs.rb

```diff
@@ -87,28 +90,27 @@ def run
               log_stream_name = log_stram.log_stream_name
               events = get_events(log_stream_name)
               events.each do |event|
-                emit(event)
+                emit(log_stream_name, event)
               end
             end
           else
             events = get_events(@log_stream_name)
             events.each do |event|
-              emit(event)
+              emit(@log_stream_name, event)
             end
           end
         end
         sleep 1
       end
     end
 
-    def emit(event)
+    def emit(stream, event)
       if @parser
         record = @parser.parse(event.message)
-        router.emit(@tag, record[0], record[1])
+        router.emit("#{@tag}.#{stream}", event.timestamp / 1000, record[1])
       else
-        time = (event.timestamp / 1000).floor
         record = JSON.parse(event.message)
-        router.emit(@tag, time, record)
+        router.emit("#{@tag}.#{stream}", event.timestamp / 1000, record)
```

Comment:
Why add `#{stream}` ?
I'd like tag to be unchanged

Reply:
+1